docs/fastdds/xml_configuration/common.rst

```diff
@@ -396,6 +389,30 @@ It contains two child elements which are explained in the following table.
 |                       | are allowed.                                              |               |                  |
 +-----------------------+-----------------------------------------------------------+---------------+------------------+
 
+.. _historymemorypoliciesXML:
+
+.. |MemManagment| replace:: :class:`MemoryManagementPolicy`
+
+
+HistoryMemoryPolicy
+^^^^^^^^^^^^^^^^^^^
+
+Indicates the way the memory is managed in terms of dealing with the CacheChanges of the :ref:`rtpsendpointqos`.
+
++----------------+--------------------------------------+------------------------------------+-------------------------+
+| Name           | Description                          | Values                             | Default                 |
++================+======================================+====================================+=========================+
+| |MemManagment| |  Four different options as described | |PREALLOCATED-xml-api|             |                         |
```

Comment:
Break the line of the description to make the table fit when rendering. Use the alias `|br|` for that|.